src/libs/antares/study/cleaner/cleaner-v20.cpp

```diff
@@ -359,7 +359,7 @@ bool listOfFilesAnDirectoriesToKeep(StudyCleaningInfos* infos)
                 logs.verbosityLevel = Logs::Verbosity::Warning::level;
                 // load all links
                 buffer.clear() << infos->folder << "/input/links/" << area->id;
-                if (not AreaLinksLoadFromFolder(*study, arealist, area, buffer))
+                if (not AreaLinksLoadFromFolder(*study, arealist, area, buffer, false))
```

Comment:
New **bool** argument should not be needed. See **links.cpp** review.

Reply:
Done !

Reply:
You're right, it would require more work.
But we should keep in mind that we don't need this **loadTSGen** fundamentally.
It's a workaround because the code has low quality.

Reply:
Yes, the loading functions are not de-coupled from the validation functions.